website/app/GeoKrety/HealthState.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace GeoKrety;
+
+class HealthState {
+    const STATE_OK = 'ok';
+    const STATE_KO = 'ko';
+    const VALID_STATES = [self::STATE_OK, self::STATE_KO];
+
+    public $state = self::STATE_OK;
+    public $dependencies = [];
```

Comment:
`protected` here too.

Reply:
json_encode need them ($state and $dependencies) to be public